src/ui/pages/Menu/Menu.tsx

```diff
@@ -96,39 +105,60 @@ const Menu = () => {
         <IonRow className="menu-input-row">
           <MenuItem
             index={0}
+            route=""
             icon={personCircleOutline}
             label={`${i18n.t("menu.tab.items.profile")}`}
           />
           <MenuItem
             index={1}
+            route=""
             icon={walletOutline}
             label={`${i18n.t("menu.tab.items.crypto")}`}
           />
         </IonRow>
         <IonRow className="menu-input-row">
           <MenuItem
             index={2}
+            route=""
             icon={peopleOutline}
             label={`${i18n.t("menu.tab.items.connections")}`}
           />
           <MenuItem
             index={3}
+            route=""
             icon={chatbubbleOutline}
             label={`${i18n.t("menu.tab.items.p2p")}`}
           />
         </IonRow>
         <IonRow className="menu-input-row">
           <MenuItem
             index={4}
+            route=""
             icon={fingerPrintOutline}
             label={`${i18n.t("menu.tab.items.identity")}`}
           />
           <MenuItem
             index={5}
+            route=""
             icon={idCardOutline}
             label={`${i18n.t("menu.tab.items.credentials")}`}
           />
         </IonRow>
+        <IonRow className="menu-input-row">
+          <MenuItem
+            index={6}
+            route={RoutePath.TUNNEL_CONNECT}
+            icon={globeOutline}
+            label={`${i18n.t("menu.tab.items.tunnel")}`}
+          />
+
+          <MenuItem
+            index={7}
+            route=""
+            icon=""
+            label=""
+          />
```

Comment:
empty card?

Reply:
Trying to avoid spending to much time on these styles, i added a extra blank card to be consistent on the menu, if not, it looks like:
![Captura de pantalla 2024-03-01 a las 12 18 47](https://github.com/cardano-foundation/cf-identity-wallet/assets/9588549/d35c4525-4381-4f6e-8c12-b1fe38e18b1b)

Reply:
ah okay fair - yeah something for Sal to fix so, it should just do flexbox correctly